piton/design/chipset/include/mc_define.h

```diff
@@ -105,6 +105,25 @@
     `define DDR3_CS_WIDTH           2
     `define DDR3_BG_WIDTH           2
     `define DDR3_ODT_WIDTH          2
+`elsif ALVEO_BOARD
```

Comment:
Are all Alveos the same? I think maybe we just duplicate this or we have the macro be ALVEO_U200_U250_BOARD

Reply:
I think the proper way to address this would be to define board under the ALVEO_BOARD macro

Reply:
Ok we can just come back to this one later when we add U280 etc